services/web/serve/env.js

```diff
@@ -0,0 +1,15 @@
+const { pick } = require('lodash');
+const config = require('@bedrockio/config');
+
+const allConfig = config.getAll();
+const PUBLIC = pick(
+  allConfig,
+  Object.keys(allConfig).filter(
+    (key) => !key.startsWith('SERVER') && !key.startsWith('HTTP')
```

Comment:
I wonder if maybe this file can't just go away entirely and instead this line (well 3 lines technically) is just in the env middleware...

Reply:
super like it